components/application_manager/utils.h

```diff
@@ -126,9 +126,8 @@ class Utils {
 
     class Timeout {
 public:
-        Timeout(std::function<void(void)> callback, std::chrono::milliseconds timeout) :
+        Timeout(std::function<void(void)> callback) :
```

Comment:
SUGGESTION - It made more sense to constuct a Timeout object with a value rather than pass in at start. You could set a default value in the arg list if you want to explicitly:
```suggestion
        Timeout(std::function<void(void)> callback, std::chrono::milliseconds timeout = Utils:AIT_TIMEOUT) :
```

Reply:
According to the spec, _If an operator application successfully launches a second operator application using createApplication, the first operator application shall be killed and the second operator application shall inherit the operator application state and the value of the countdown timer (see below), when applicable, from the first operator application._
So there was a need to move the timeout value to the start() method and when an OpApp creates another OpApp, it would pass its remaining timeout to the start() method of the new app.

Reply:
Make sense.